src/runtimes/docker/utils/normalizeContainerOS.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Return a normalized string for the OS name specifically for Docker
+ * @param os pre-normalized OS name
+ * @returns normalized OS name
+ */
+export function normalizeContainerOS(os?: string): string {
+    os = os?.toLowerCase() || 'linux'; // default to linux if not specified (null/undefined or empty string)
+    switch (os) {
```

Comment:
We may want to handle 'win' to 'windows' as well as 'osx' to 'darwin' for those old school Mac users who haven't gotten used to the OS rebranding.

Reply:
did not know that this was a thing, good suggestion